addon/components/dashboard/courses-calendar-filter.hbs

```diff
@@ -1,6 +1,7 @@
 <div
   class="calendar-filter-list large-filter-list dashboard-courses-calendar-filter"
-  {{did-insert (perform this.load) @school}}
+  {{did-insert (set this.el)}}
+  {{did-insert (perform this.load)}}
```

Comment:
pass the school to the `load()` task on insert as well  `{{did-insert (perform this.load)}}`.